src/main/java/io/vertx/ext/mongo/impl/Utils.java

```diff
@@ -82,6 +89,22 @@ private static JsonObject convertUpsertId(BsonValue upsertId) {
     return jsonUpsertId;
   }
 
+  private static JsonObject convertInsertId(BsonValue inserId) {
```

Comment:
Instead of duplicating the method, we should rename `convertUpsertId` to `convertWriteResultId` or something similar.